packages/ui/components/proposal/ProposalYourVote.tsx

```diff
@@ -1,5 +1,27 @@
-export interface ProposalYourVoteProps {}
+import clsx from 'clsx'
 
-export const ProposalYourVote = ({}: ProposalYourVoteProps) => {
-  return <p>placeholder</p>
+export type YourVoteProps = {
+  variant: 'abstain' | 'pending' | 'no' | 'yes'
+}
+
+export const ProposalYourVote = ({ variant }: YourVoteProps) => {
+  return (
+    <div
+      className={clsx(
+        'flex relative justify-center items-center py-1 px-4 w-20 text-sm font-medium rounded-full',
+        variant === 'abstain'
+          ? 'text-text-secondary bg-primary'
+          : variant === 'pending'
+          ? 'border-2 border-inactive'
+          : variant === 'no'
+          ? 'text-text-interactive-error bg-[#C73E5914]'
```

Comment:
```suggestion
          ? 'text-text-interactive-error bg-background-interactive-error'
```

`#C73E59` corresponds to `rgb(199, 62, 89)`, which exists in the colors as `color-error`. Since tailwind lets you change the opacity of colors
([like this](https://tailwindcss.com/docs/background-color#changing-the-opacity)), this doesn't need to be hardcoded, so `bg-color-error/[0.08]` should work.

Alternatively, we can just use `bg-background-interactive-error` since it uses the error color with an opacity of `0.1`, which is negligibly different.

Reply:
hey the change is pretty notice-able on dark mode. but we can fix this later. Thanks for the suggestions!